src/algebra/module/equiv.lean

```diff
@@ -225,7 +222,13 @@ include σ'
 @[simp] lemma inv_fun_eq_symm : e.inv_fun = e.symm := rfl
 omit σ'
 
-@[simp] lemma coe_to_equiv_symm : ⇑e.to_equiv.symm = e.symm := rfl
+@[simp, norm_cast] lemma coe_to_equiv_symm : (e.symm : M₂ ≃ M) = (e : M ≃ M₂).symm := rfl
+
+@[simp, norm_cast]
+lemma coe_to_add_equiv_symm : (e.symm : M₂ ≃+ M) = (e : M ≃+ M₂).symm := rfl
+
+@[simp] lemma coe_to_equiv_symm_apply (x : M₂) : (e : M ≃ M₂).symm x = e.symm x := rfl
+@[simp] lemma coe_to_add_equiv_symm_apply (x : M₂) : (e : M ≃+ M₂).symm x = e.symm x := rfl
```

Comment:
You can state these as equalities between two functions without mentioning `(x : M₂)`.